src/uproot_browser/plot_view.py

```diff
@@ -23,8 +24,14 @@
 def make_plot(item: Any, *size: int) -> Any:
     plt.clf()
     plt.plotsize(*size)
-    plt.title("Plotext Integration in Rich - Test")
-    uproot_browser.plot.plot(item)
+    h = uproot_browser.plot.plot(item)
+    if (np.sum(h.values())) == (np.sum(h.values(flow=True))):
```

Comment:
```suggestion
    if math.isclose(np.sum(h.values()), np.sum(h.values(flow=True))):
```

+ `import math`

Might help for floating point / weighted storage.